modules/tracktion_engine/plugins/external/tracktion_ExternalPlugin.cpp

```diff
@@ -1707,22 +1726,38 @@ bool ExternalPlugin::setBusLayout (juce::AudioChannelSet set, bool isInput, int
 }
 
 //==============================================================================
-juce::String ExternalPlugin::createPluginInstance (const juce::PluginDescription& description)
+void ExternalPlugin::createPluginInstance (const juce::PluginDescription& description)
 {
     jassert (! pluginInstance); // This should have already been deleted!
 
     auto& dm = engine.getDeviceManager();
-
-    juce::String error;
-    pluginInstance = engine.getPluginManager().createPluginInstance (description, dm.getSampleRate(), dm.getBlockSize(), error);
-
-    if (pluginInstance != nullptr)
+    
+    if (requiresAsyncInstantiation (engine, description))
+    {
+        // While instantiation is pending, make sure the plugin is disabled
+        setEnabled (false);
+        engine.getPluginManager().pluginFormatManager
+          .createPluginInstanceAsync (description, dm.getSampleRate(), dm.getBlockSize(),
+                                      [&] (auto _instance, const auto& _err) -> void {
```

Comment:
Just a small style thing, we don't use leading underscores for name.
As the standard can reserve anything starting with an underscore in the global namespace (include `#define` etc) these can be problematic. Especially if they have generic names like `_err`. I'd just remove the leading underscores here.